olog/httpx_client.py

```diff
@@ -42,38 +56,84 @@ async def aget_logbook(self, name):
         async with self._session as api:
             res = await api.get(f'logbooks/{name}')
         res.raise_for_status()
-        return res.json()
+        return simplify_logbook(res.json())
 
     def get_logbook(self, name):
+        '''
+        Parameters
+        ----------
+        name : str
+            name of logbook
+
+        Returns
+        -------
+        logbook : dict
+            Each element is a logbook dict.
+
+        Examples
+        --------
+        >>>cli.get_logbook("TEST')
+        {'TEST': {'owner': 'olog-logs', 'state': 'Active'}}
+
+        '''
         return asyncio.run(self.aget_logbook(name))
 
-    async def aput_logbooks(self, logbooks):
+    async def aput_logbooks(self, names):
```

Comment:
Is no other content by a name and owner allowed?

Reply:
owner has to be current user(`self.user`). Also server will put `state`  value into dict. So client actually could only give names.